### PR TITLE
Change AFU interface type's key from "name" to "class".

### PR DESCRIPTION
**Pull request by michael-adler**
Using "name" was a source of confusion every time I explained the PIM's AFU
JSON.  They would see "afu-top-interface:name" and think that's the module
name, which was a reasonable assumption.  It isn't.  Instead, it is the class
of the interface, defining the ports and not the module name.

For compatibility, the legacy key "name" is accepted as a synonym.

### CODE REVIEW

**Comment by michael-adler** [comment]
Tested both old JSON (using "name") and new JSON (using "class"). Both work:
http://sw-pert.altera.com/pert/pert.php?test_run_id=2273220
http://sw-pert.altera.com/pert/pert.php?test_run_id=2273301

File: platforms/afu_top_ifc_db/ccip_std_afu.sv.template [hunk 1]
```diff
@@ -28,7 +28,7 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-import ccip_if_pkg::*;
```
Comment (r-rojo) [inline platforms/afu_top_ifc_db/ccip_std_afu.sv.template]: Is this change related to the `name` => `class` change?

File: platforms/afu_top_ifc_db/ccip_std_afu_avalon_mm.sv.template [hunk 2]
```diff
@@ -28,7 +28,7 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-`include "device_if.vh"
```
Comment (r-rojo) [inline platforms/afu_top_ifc_db/ccip_std_afu_avalon_mm.sv.template]: Is this change related to the name => class change?

File: platforms/afu_top_ifc_db/ccip_std_afu_avalon_mm_legacy_wires.sv.template [hunk 3]
```diff
@@ -28,7 +28,7 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-import ccip_if_pkg::*;
```
Comment (r-rojo) [inline platforms/afu_top_ifc_db/ccip_std_afu_avalon_mm_legacy_wires.sv.template]: Is this change related to the name => class change?

File: platforms/afu_top_ifc_db/ccip_std_afu.sv.template (hunk 1 above)
Comment (michael-adler) [reply platforms/afu_top_ifc_db/ccip_std_afu.sv.template]: It's related cleanup. The AFU needs platform_if.vh to function properly.

File: platforms/afu_top_ifc_db/ccip_std_afu_avalon_mm.sv.template (hunk 2 above)
Comment (michael-adler) [reply platforms/afu_top_ifc_db/ccip_std_afu_avalon_mm.sv.template]: Same here.

File: platforms/afu_top_ifc_db/ccip_std_afu_avalon_mm_legacy_wires.sv.template (hunk 3 above)
Comment (michael-adler) [reply platforms/afu_top_ifc_db/ccip_std_afu_avalon_mm_legacy_wires.sv.template]: And here.
